posts/2023-02-20-aws-instant-on.adoc

```diff
@@ -0,0 +1,223 @@
+```
+
+=== Deploy the InstantOn application
+
+After the cluster is up and running, we can deploy the Liberty InstantOn application. 
+
+First, create a `deployment.yaml` file with the following contents. Update the `image` value to match the `repositoryUri` from the `create-repository` command output.
+
+.deployment.yaml
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: open-liberty-instanton
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: open-liberty-instanton
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: open-liberty-instanton
+    spec:
+      containers:
+      - image: <aws_account_id>.dkr.ecr.<region>.amazonaws.com/getting-started-instanton
+        imagePullPolicy: IfNotPresent
+        name: app
+        ports:
+        - containerPort: 9080
+          name: 9080-tcp
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 1
+            memory: 512Mi
+          requests:
+            cpu: 500m
+            memory: 256Mi
+        securityContext:
+          runAsNonRoot: true
+          privileged: false
+          capabilities:
+            add:
+            - CHECKPOINT_RESTORE
+            - SETPCAP
+            drop:
+            - ALL
+----
+
+
+Next, deploy the application by running the following command:
+```
+kubectl apply -f deployment.yaml
+```
+
+Check the logs to see if the application started up successfully. Keep in mind that it might take a few extra seconds for the pod to start for very first time as the cluster nodes must pull down the container image from ECR:
+```
+kubectl logs -l app.kubernetes.io/name=open-liberty-instanton --tail=-1
+```
+
+Look for the following output to confirm that the application started successfully with InstantOn:
+```
+[AUDIT   ] CWWKZ0001I: Application io.openliberty.sample.getting.started started in 0.331 seconds.
+[AUDIT   ] CWWKC0452I: The Liberty server process resumed operation from a checkpoint in 0.464 seconds.
+[AUDIT   ] CWWKF0012I: The server installed the following features: [cdi-2.0, checkpoint-1.0, distributedMap-1.0, jaxrs-2.1, jaxrsClient-2.1, jndi-1.0, json-1.0, jsonp-1.1, monitor-1.0, mpConfig-2.0, mpHealth-3.1, mpMetrics-3.0, servlet-4.0, ssl-1.0].
+[AUDIT   ] CWWKF0011I: The defaultServer server is ready to run a smarter planet. The defaultServer server started in 0.510 seconds.
+[AUDIT   ] CWWKS4104A: LTPA keys created in 0.919 seconds. LTPA key file: /opt/ol/wlp/output/defaultServer/resources/security/ltpa.keys
+[AUDIT   ] CWPKI0803A: SSL certificate created in 3.290 seconds. SSL key file: /opt/ol/wlp/output/defaultServer/resources/security/key.p12
+```
+
+Now, you can scale the application up to see how quickly the new pod instances are coming up! Results may vary but you should see up to 90% improvement in the startup time.
```

Comment:
> Results may vary but you should see up to 90% improvement in the startup time.

Improvement over what? I might be reading this wrong, but is there anything to compare the startup time to here? Is it compared to starting the Liberty starter w/out InstantOn?

Reply:
updated.